### DISCUSSION TITLE
Expand Latin-M?

### DISCUSSION THREAD

**Discussion by jpt** [koeberlin/Latin-Character-Sets#18] [Ideas]
Through Justin Penner's Talking Leaves plugin, I noticed there are around ~2MM additional readers who can be added to Latin-M "for free," as in from existing combining marks. Does it make sense to add them, or is there a specific criteria that Latin-M has in mind for additions to meet? I can find the specific orthographies and glyphs if you have any interest. Cheers

**Comment by koeberlin** [comment]
Hi Jeremy, would love to hear!

**Comment by jpt** [comment]
Great! These are the groups that seem totally free:

Kara-Kalpak, 583,410 speakers: Gacute gacute (Ǵ ǵ)
Ewondo, 580,000 speakers, as well as Basa (Cameroon), 300,000 speakers, and a few others below 100k: Ngrave ngrave (Ǹ ǹ)
Mundang, 240,000 speakers: Vtilde vtilde (Ṽ ṽ)

There are a few others that require _some_ effort but not a lot - and some that may or may not autogenerate in font editors, need to test. I'm less sure if these would make it into Latin-M, basically. These include:

Venda, 1,300,000 speakers: Dcircumflexbelow dcircumflexbelow Lcircumflexbelow lcircumflexbelow Ncircumflexbelow ncircumflexbelow Tcircumflexbelow tcircumflexbelow — people may not already have a circumflexbelow, does it auto generate in Glyphs maybe?
Hassaniyya, 3,200,000 speakers: Xdotabove xdotabove - these are free free, but Tbar tbar - not quite free (Ẋ ẋ ŧ Ŧ)
Lyele, 130,000 speakers: Otildeacute otildeacute Ṍṍ - again, do people have tildeacute and/or does it autocompose?
Nateni, 66,000 speakers:  Iacutebelow iacutebelow Uacutebelow uacutebelow (Ḭ ḭ Ṵ ṵ) - do people have acutebelow/does it autocompose?
These are  eported as missing for Mandarin transliteration - Ẑ Ḿ ḿ ẑ - but I don't know if transliteration is a goal of Latin-M

This may not be exhaustive, and I'm no expert on whether there are any considerations beyond adding the glyphs, or whether Hyperglot (which Talking Leaves sources its data from) is accurate to how these are used in the real world

**Reply by koeberlin** [reply]
Hey Jeremy, thanks again for your suggestion! 
I thought about it, but in the end I decided for a different solution. I noticed that once you have Vietnamese support, your half way to supporting DIN 91379, which is requested more often lately. So I thought that made a nice and consistent intermediate package.

**Reply by jpt** [reply]
> Hey Jeremy, thanks again for your suggestion! I thought about it, but in the end I decided for a different solution. I noticed that once you have Vietnamese support, your half way to supporting DIN 91379, which is requested more often lately. So I thought that made a nice and consistent intermediate package.

I appreciate the update - if an existing standard already covers things, that sounds like a great idea to utilize it :)
